dbt/include/databricks/macros/materializations/snapshot.sql

```diff
@@ -102,10 +106,6 @@
 
   {{ adapter.commit() }}
 
-  {% if staging_table is defined %}
-      {% do post_snapshot(staging_table) %}
```

Comment:
The default implementation of `post_snapshot` seems to be a noop 
https://github.com/dbt-labs/dbt-core/blob/e7218d3e99837f0139fb7ecd367d3bdf1135a961/core/dbt/include/global_project/macros/materializations/snapshots/helpers.sql#L17-L23
And dbt-spark doesn't seem to implement it either. Where is the logic of post_snapshot defined?

Reply:
`dbt-spark` does have the implementation. 

https://github.com/dbt-labs/dbt-spark/blob/main/dbt/include/spark/macros/materializations/snapshot.sql#L52-L54

Reply:
:+1: This doesn't show up in Github's search result..